autoextract_spiders/spiders/autoextract_article.py

```diff
@@ -10,6 +10,17 @@ class ArticleAutoExtract(CrawlerSpider):
     name = 'articles'
     page_type = 'article'
 
+    frontera_settings = {
+        'HCF_PRODUCER_FRONTIER': 'autoextract',
```

Comment:
We might have some reasonable defaults here in a case if user doesn't want to bother with details. Just enabling Frontera/HCF with the setting already brings some significant benefits.